turbinia/client.py

```diff
@@ -895,6 +904,16 @@ def __init__(self, jobs_blacklist=None, jobs_whitelist=None):
       jobs_whitelist (Optional[list[str]]): The only Jobs we will include to run
     """
     super(TurbiniaCeleryWorker, self).__init__()
+
+    if jobs_whitelist or jobs_blacklist:
+      selected_jobs = jobs_blacklist or jobs_whitelist
+      for job in selected_jobs:
+        if job.lower() not in TASK_MAP:
```

Comment:
I dont' think this will work because this is the list of Tasks, not the list of Jobs.  I think you'll have to check the Job manager.